reactioncommerce/reaction
# [WIP] refactor: remove `coreOrderItemWorkflow

Resolves #issueNumber  
Impact: **major**  
Type: **refactor**

## Issue
Items are managed by fulfillment groups. Since the group has a status on it, we don't also need a status on each item.

## Solution
Remove `coreOrderItemStatus` from the `Order.item` schema, and remove all code related to updating this status. Where applicable, change status checks to check group status.

## Testing
1. Make sure you can create an order
1. Move the order through various shipment statuses (cancel, shipped, completed, etc)
1. See that everything works as it should
